sdk/core/azure_core_test/src/tracing.rs

```diff
@@ -252,21 +316,50 @@ pub fn check_instrumentation_result(
         assert_eq!(tracer.namespace, expected.namespace);
 
         let spans = tracer.spans.lock().unwrap();
-        assert_eq!(
-            spans.len(),
-            expected.spans.len(),
-            "Unexpected number of spans for tracer {}",
-            expected.name
-        );
 
-        for (span_index, span_expected) in expected.spans.iter().enumerate() {
+        // assert_eq!(
+        //     spans.len(),
+        //     expected.spans.len(),
+        //     "Unexpected number of spans for tracer {}",
+        //     expected.name
+        // );
+
+        let mut expected_index = 0;
+        for (span_index, span_actual) in spans.iter().enumerate() {
             eprintln!(
                 "Checking span {} of tracer {}: {}",
-                span_index, expected.name, span_expected.span_name
+                span_index, expected.name, span_actual.name
```

Comment:
Was this just wrong before and this fixed that? It's an unexpected change otherwise.

Reply:
The trace that was generated was just wrong.